docs/contribute.rst

```diff
@@ -0,0 +1,105 @@
+.. include:: ../.github/CONTRIBUTING.rst
+
+
+First-time setup
```

Comment:
Mention somewhere that docs need Python 3.

Reply:
@webknjaz Sorry, Python3 to need for that?
To build docs?
But it just runs tox command.
Don't understand this moment.
Where do I need to write what to need Python3?